cmd/consensus/main.go

```diff
@@ -614,36 +614,6 @@ func main() {
 
 			return aggregator, nil
 		}).
-		Module("compliance engine", func(builder cmd.NodeBuilder, node *cmd.NodeConfig) error {
```

Comment:
modules are initialized before components, but compliance engine initialized depends on `hotstuffModules` which can be initialized only as component(because it uses network which is a component)